lang/lang.pt-BR.json

```diff
@@ -765,13 +765,13 @@
     "refs": {
         "meta": {
             "language": "pt-BR",
-            "languageDisplay": "Portugês",
+            "languageDisplay": "Português Brasileiro",
             "locale": "pt-BR",
-            "translators": "KiddLankaster#2198"
+            "translators": "KiddLankaster#2198, [Davi 6596](https://github.com/Davi-the-Mudkip)"
```

Comment:
Do you want your full discord name here? "Davi 6596#6596"?

Reply:
Yeah, I forgot to add the tag. Thanks.

Reply:
Done.